include/NovelRT.Interop/DotNet/NrtDotNetTypedefs.h

```diff
@@ -1,19 +1,18 @@
 // Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
 // for more information.
 
-#include <NovelRT.Interop/NrtInteropErrorHandlingInternal.h>
-#include <NovelRT.Interop/NrtInteropUtils.h>
```

Comment:
What happened here? how did includes get replaced with an include guard?

Reply:
This is GitHub seeing a delete and file addition where the contents between the two files is similar enough it wants to show the diff as a rename.

The old file was removed (and merged into `NrtTypedefs.h`) and the new file is completely different/unrelated.